server/meshmodel/policies/validator.json

```diff
@@ -0,0 +1,27 @@
+{
+    "kind":"Validation",
+    "subType":"rjsf",
+    "apiVersion":"core.meshery.io/v1alpha1",
+    "model":{
+        "name":"kubernetes"
```

Comment:
@harkiratsm Add version and other fields.

Reply:
Done :heavy_check_mark: